Modules/Office365DSC/DSCResources/MSFT_EXOAntiPhishRule/MSFT_EXOAntiPhishRule.schema.mof

```diff
@@ -0,0 +1,17 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOAntiPhishRule")]
+class MSFT_EXOAntiPhishRule : OMI_BaseResource
+{
+    [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
+    [Key, Description("The Identity parameter specifies the antiphishing rule that you want to modify.")] String Identity;
```

Comment:
Please provide description for every parameter

Reply:
Descriptions have been populated.